Core/clim-core/frames/define-application-frame.lisp

```diff
@@ -86,7 +86,21 @@
      ;; Update frame-current-panes and the special pane slots.
      (update-frame-pane-lists frame)))
 
+(defun geometry-specification-p (thing)
+  (and (alexandria:proper-list-p thing)
+       (evenp (length thing))
+       (loop for (key value) on thing by #'cddr
+             always (member key '(:left :top :right :bottom :width :height)))))
+
 (defun parse-define-application-frame-options (options)
+  ;; Note that options are always of the form (KEY
+  ;; . ARGUMENTS). Depending on KEY, ARGUMENTS is expected to have one
```

Comment:
this line wrap seems bogus

Reply:
Must be Emacs getting confused by the the `.`. Fixed.